.github/workflows/overcloud-host-image-build.yml

```diff
@@ -94,11 +94,15 @@ jobs:
         if: inputs.centos
 
       - name: Upload CentOS Stream 8 overcloud host image artifact
-        uses: actions/upload-artifact@v3
-        with:
-          name: CentOS Stream 8 overcloud host image
-          path: /opt/kayobe/images/overcloud-centos-stream-8
-          retention-days: 7
+        run: |
+          kayobe playbook run \
+          src/kayobe-config/etc/kayobe/ansible/upload-image.yml \
+          -i localhost \
+          -e image_path='/opt/kayobe/images/overcloud-centos-stream-8' \
+          -e os_distribution='centos' \
+          -e os_version='stream8'
```

Comment:
I think the "official" supported value of os_version is `8-stream`. See https://docs.openstack.org/kayobe/yoga/configuration/reference/os-distribution.html

Reply:
Kayobe should set a sensible default based on os_distribution.

Reply:
Shall we try omitting it then?

Reply:
ok. The image names might need to be updated to match though

Reply:
also, should it be os_release rather than os_version?

Reply:
ah, I hadn't spotted that you used a different variable. Let's try to use os_release then